webapp/channels/src/actions/views/rhs.test.ts

```diff
@@ -109,7 +110,7 @@ describe('rhs view actions', () => {
                         first_name: currentUserFirstName,
                         timezone: {
                             useAutomaticTimezone: true,
-                            automaticTimezone: '',
+                            automaticTimezone: 'UTC+3', // set an arbitrary timezone
```

Comment:
Having no timezone messes with the tests based on which timezone it's running. It's likely happening since the UTC fallback was added in this PR https://github.com/mattermost/mattermost/pull/27301/files#diff-8d056d1255298c176eec332ec3a43caa37bef1d3eef9819436ffdfc2db63c984R24

Reply:
And thanks for adding this! Given how annoying flaky tests are, I definitely appreciate anything we can do to make them less flaky